AutoComplete List - Fall Documentation 2019 (#138)
smartdevicelink/sdl_app_library_guides: docs/Displaying a User Interface/Popup Menus and Keyboards/index.md

}

/// Optional Methods
- (void)updateAutocompleteWithInput:(NSString *)currentInputText autoCompleteResultsHandler:(SDLKeyboardAutoCompleteResultsHandler)resultsHandler {

>     <#Check the input text and return an array of string that contain the current input text#>

Reviewer: ```suggestion
    <#Check the input text and return an array of autocomplete results#>
```

>     if ([currentInputText.lowercaseString hasPrefix:@"f"]) {

Reviewer: ```suggestion
```

>         resultsHandler(@[@"First", @"Friend", @"Fruit"]);

Reviewer: ```suggestion
    resultsHandler(@[<#String results to be displayed#>]);
```

>     }

Reviewer: ```suggestion
```

    }

    /// Optional Methods
    func updateAutocomplete(withInput currentInputText: String, autoCompleteResultsHandler resultsHandler: @escaping SDLKeyboardAutoCompleteResultsHandler) {

>         <#Check the input text and return an array of string that contain the current input text#>

Reviewer: ```suggestion
        <#Check the input text and return an array of autocomplete results#>
```

>          if currentInputText.lowercased().hasPrefix("f") {

Reviewer: ```suggestion
```

>             resultsHandler(@["First", "Friend", "Fruit"]);

Reviewer: ```suggestion
        resultsHandler([<#String results to be displayed#>]);
```

>         }

Reviewer: ```suggestion
```
